docs/tutorial.rst

```diff
@@ -0,0 +1,88 @@
+.. _tutorial:
+
+Tutorial
+========
+
+This is a step-by-step tutorial to help you configure OnionBalance.
+
+OnionBalance implements `round-robin` like load balancing on top of Tor
+onion services. A typical OnionBalance deployment will incorporate one management
+servers and multiple backend application sdoervers.
+
+Assumptions
+-----------
+
+You want to run:
+
+- one or more OnionBalance processes, to perform load balancing, on hosts named obhost1, obhost2
+- two or more Tor processes, to run the Onion Services, on hosts named torhost1, torhost2
+- two or more servers (e.g. web servers) on hosts named webserver1, webserver2
+
+Scaling up:
+
+- the number of obhostX can be increased by this will not help handling more traffic
+- the number of torhostX can be increased up to 10 instances to handle more traffic
+- the number of webserverX can be increased up to 60 to handle more traffic
```

Comment:
@FedericoCeratto I'm not sure the distinction between the Tor host and webserver here. You can scale the HTTP traffic after Tor with a standard application level proxy such as haproxy.

OnionBalance can now use introduction points from 60 different Tor instances at one time. There can be 60 torhostX hosts and there is no limit on the number of web servers.